### DISCUSSION TITLE
What does  Invoke-WebRequest -Uri $cutt_url do?

### DISCUSSION THREAD

**Discussion by spacecat** [SpotX-Official/SpotX#179] [Question]
### Description of the new feature / enhancement

$cutt_url = "https://cutt.ly/DK8UQub"

This url resturns "test".

But does it really?

**Comment by amd64fox** [comment]
[Cutt.ly](https://cutt.ly/) is just a service for shortening links and providing statistics, this shortened link will enter the file https://raw.githubusercontent.com/SpotX-CLI/SpotX-Win/main/.github/cutt/test-cutt.txt

That is, in simple words, this web request, which provides general data about users who run the installation of SpotX, for example:

![Снимок экрана 2022-10-08 174644](https://user-images.githubusercontent.com/62529699/194713228-20f704e4-2f67-44d6-a280-92e742223a21.jpg)
![Снимок экрана 2022-10-08 174620](https://user-images.githubusercontent.com/62529699/194713230-448e7fc1-d035-4a2c-8a4d-a7131ac8050c.jpg)
![Снимок экрана 2022-10-08 174527](https://user-images.githubusercontent.com/62529699/194713231-2f1d9803-7a4a-4976-a09e-623f80c0b4e8.jpg)

Basically, this is all the statistics that I see, perhaps it is in a more advanced form, but for this you need to buy a cutt subscription, a free account is enough for me.

**Reply by spacecat** [reply]
Okay I think I know how it works and what it does now. Thanks for clarifying 👍🏼
